__tests__/__fixtures__/single-file-todo-updated.json

```diff
@@ -25,9 +25,9 @@
         "endColumn": 33
       },
       {
-        "ruleId": "no-prototype-builtins",
+        "ruleId": "no-unused-vars",
         "severity": 2,
-        "message": "Do not access Object.prototype method 'hasOwnProperty' from target object.",
+        "message": "'foo1' is assigned a value but never used.",
```

Comment:
It was necessary to change this file since it's expected to be used in a test that requires an updated lint violation, and this violation was matching due to fuzzy matches.